Gosu/Image.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#ifdef __cplusplus
+  extern "C" {
+#endif
+
+typedef struct Gosu_Image Gosu_Image;
+
+typedef struct Gosu_GLTexInfo {
+  int text_name;
```

Comment:
I think `tex_name` or `texture_name` would be less misleading than `text_name`

Reply:
Went with `texture_name` as it seems the most logical